binr/rabin2/rabin2.c

```diff
@@ -937,6 +937,8 @@ int main(int argc, char **argv) {
 				break;
 			}
 		}
+		// if we reach here, no supported xtr plugins found
```

Comment:
this message will be shown even if the file is extracted properly. the break will only quit the r_list_foreach block, not the whole function